src/containers/movie_list.js

```diff
@@ -56,8 +58,12 @@ class MovieList extends Component{
     if(this.props.search.term !== null){
       return (
         <div>
-          <h1>Results for "{this.props.search.term}"</h1>
-          {this.props.movies.map(this.renderMovie)}
+          <h1 className="page-title">
```

Comment:
@igorcmoura I've created an class ours for titles
![title-size](https://cloud.githubusercontent.com/assets/3057676/26516507/8bcfc312-425d-11e7-8073-aef86c734514.png)